src/page/ChecksPage.tsx

```diff
@@ -5,7 +5,7 @@ import React, { PureComponent } from 'react';
 import { Check } from 'types';
 import { getLocationSrv } from '@grafana/runtime';
 import { CheckEditor } from 'components/CheckEditor';
-import { CheckList } from 'components/CheckList';
+import { CheckList } from 'components/CheckList/CheckList';
```

Comment:
You could put an `index.ts` that manages the exports from that directory if you like so the imports don't have to change